ajax/libs/console.js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "console.js",
+  "filename": "console.min.js",
+  "version": "0.0.2",
+  "description": "A sane way to interface with your browser console and use the latest features.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/icodeforlove/console.js.git"
+  },
+  "homepage": "http://icodeforlove.github.io/Console.js/",
```

Comment:
@Piicksarn It will be better to use `https` instead of `http` for `github.io` domain.